tests/optimizer/minimal-runtime-applyDCEGraphRemovals.js

```diff
@@ -0,0 +1,24 @@
+var name;
+var asmLibraryArg = { 'save1': 1, 'number': 33, 'name': name, 'func': function() {}, 'save2': 2 };
+
+// exports gotten directly in the minimal runtime style
+WebAssembly.instantiate(Module["wasm"], imports).then(function(output) {
+ asm = output.instance.exports;
+ expD1 = asm['expD1'];
+ expD2 = asm['expD2'];
+ expD3 = asm['expD3'];
+ expD4 = asm['expD4'];
+ initRuntime(asm);
+ ready();
+});
+
+// add uses for some of them, leave *4 as non-roots
+expD1;
+Module['expD2'];
+asm['expD3'];
+
+expI1;
+Module['expI2'];
+asm['expI3'];
```

Comment:
Hmm, do these `expI*` intentionally not exist?

Reply:
They made more sense in the non-minimal-runtime test where there is another way to export. I just copied the test input and modified it to the minimal extent, to make comparisons easier. You're right though that it makes reading the file by itself slightly confusing. I'll remove them.